src/components/MovieCard/MovieCard.tsx

```diff
@@ -3,7 +3,7 @@ import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
 type Props = {
-  movie: Movie,
+  movie: Movie | null,
```

Comment:
```suggestion
  movie: Movie,
```
since you have conditional rendering in the `FindMovie` component it's redundant